test/execute/main.js

```diff
@@ -1626,6 +1627,95 @@ describe('execute', () => {
         })
       })
 
+      it('should handle duplicate parameter inheritance from normalized swagger specifications', function () {
+        const spec = {
+          spec: {
+            host: 'swagger.io',
+            basePath: '/v1',
+            paths: {
+              '/pet/{id}': {
+                parameters: [
+                  {
+                    name: 'id',
+                    in: 'path',
+                    type: 'number',
+                    required: true
+                  }
+                ],
+                get: {
+                  operationId: 'getPetsById',
+                  parameters: [
+                    {
+                      name: 'test',
+                      in: 'query',
+                      type: 'number'
+                    }
+                  ],
+                }
+              }
+            }
+          }
+        }
+
+        const resultSpec = normalizeSwagger(spec)
```

Comment:
the existing unit tests were not catching this condition because none of the tests normalizeSwagger before calling buildRequest.

Reply:
Now we have such a test, and we have a regression test verifying that the warning is not thrown in this situation where the parameters are inherited by the operation.